_layouts/login.html

```diff
@@ -28,26 +28,26 @@ <h1 class="pf-c-title pf-m-3xl">Log in to your account</h1>
               </div>
               <div class="pf-c-form__group">
                 <label class="pf-c-form__label" for="inputUsername">
-                  <span class="pf-c-form__label-text">Username</span>
+                  <span class="pf-c-form__label-text">{{ .Locale.Username }}</span>
                   <span class="pf-c-form__label-required" aria-hidden="true">*</span>
                 </label>
                 <input type="text" class="pf-c-form-control" id="inputUsername" placeholder="" tabindex="1" autofocus="autofocus" type="text" name="{{ .Names.Username }}" value="{{ .Values.Username }}">
               </div>
               <div class="pf-c-form__group">
                 <label class="pf-c-form__label" for="inputPassword">
-                  <span class="pf-c-form__label-text">Password</span>
+                  <span class="pf-c-form__label-text">{{ .Locale.Password }}</span>
                   <span class="pf-c-form__label-required" aria-hidden="true">*</span>
                 </label>
                 <input type="password" class="pf-c-form-control" id="inputPassword" placeholder="" tabindex="2" type="password" name="{{ .Names.Password }}" value="">
               </div>
               <div class="pf-c-form__group pf-m-action">
-                <button class="pf-c-button pf-m-primary pf-m-block" type="submit" tabindex="3">Log in</button>
+                <button class="pf-c-button pf-m-primary pf-m-block" type="submit" tabindex="3">{{ .Locale.LogIn }}</button>
               </div>
             </form>{% endraw %}
           </div>
         </main>
         <footer class="pf-c-login__footer">
-          <p>Welcome to {{ collection.name }}.</p>
+          <p>{{ .Locale.WelcomeTo }} {{ collection.name }}.</p>
```

Comment:
```suggestion
          <p>{% raw %}{{ .Locale.WelcomeTo }}{% endraw %} {{ collection.name }}{% raw %}{{ .Locale.WelcomeToEnd }}{% endraw %}</p>
```
But this leaves the product name in english.  Not sure if we want that or want to translate it.  The logo above is in english, so maybe it's ok?